Makefile

```diff
@@ -30,18 +30,7 @@ ifdef DB
 endif
 	-php bin/console doctrine:database:drop --force --env=test
 	php bin/console doctrine:database:create --env=test
-ifndef DB ## make test does not define DB
```

Comment:
Please keep in mind that some migrations don't support SQLite (`Version20161214094402`, `Version20170511211659`). So we could have inconsistencies with `make test`

Reply:
There was 2 migrations not executed correctly `php bin/console doctrine:migrations:migrate ` and I need modify it, but `Version20161214094402` and `Version20170511211659` were OK, it seems that only the `down` doesn't support SQLITE

Reply:
But I don't understand why in local SQLite migration was not OK, but with travis it was OK

Reply:
Ok in travis the versions not correct in local was skipped

```
  ++ migrating 20170824113337

  SS skipped (Reason: It seems that you already played this migration.)
  
  ++ migrating 20160812120952

  SS skipped (Reason: It seems that you already played this migration.)
```